### PR TITLE
Feat/web 3/splash section

### PR DESCRIPTION
**Pull request by mck-jacky**
finished implementation of Splash Section

### CODE REVIEW

**Comment by mck-jacky** [comment]
finished implementation of Splash Section with new file structure

File: src/routes/LandingPage/SplashSection/index.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.SplashBlurb {
+    font-size: 1rem;
+    color: rgba(25, 4, 49, 1);
+    max-width: 35%;
+    margin-bottom: 100px;
+    margin-left: 100px;
+    text-align: left;
+}
+
+.SplashButton {
```
Comment (imagine-hussain) [inline src/routes/LandingPage/SplashSection/index.scss:17]: https://stackoverflow.com/questions/11497094/remove-border-from-buttons

Removing that default border would really clean this up

**Review by imagine-hussain — CHANGES_REQUESTED** [review]
There's a lot of small things here that could be a little differently (most of it is just convention stuff, the code is good) as you'll see in the comments on code but, overall, really well done!

I've just got a question about the `crop_logo_circle.png`. How was that generated?

 
==========
Img for my own reference
image

**Comment by mck-jacky** [comment]
> There's a lot of small things here that could be a little differently (most of it is just convention stuff, the code is good) as you'll see in the comments on code but, overall, really well done!
> 
> I've just got a question about the `crop_logo_circle.png`. How was that generated?
> 
> ========== Img for my own reference image

I just downloaded the icon and cropped it by myself

**Comment by InvalidHero** [comment]
It looks a bit off
- button styling can maybe be changed
- positioning seems a bit off, e.g. the button should maybe flex till the right side of the screen @imagine-hussain ?

**Comment by imagine-hussain** [comment]
> It looks a bit off
> 
> - button styling can maybe changed
> 
> - positioning seems a bit off, e.g. the button should maybe flex till the right side of the screen @imagine-hussain ?

So the positioning is dependent on your screen width so it looks good sometimes and a bit off at other times. I'm happy to merge this as is though, and then leave the responsiveness to next sprint. But if @mck-jacky wants to address this, they Sgtm too.

The button styling I think looks off mainly cos of the default border that's still lingering around.
@InvalidHero

**Review by imagine-hussain — APPROVED** [review]

**Comment by mck-jacky** [comment]
> > It looks a bit off
> > 
> > * button styling can maybe changed
> > * positioning seems a bit off, e.g. the button should maybe flex till the right side of the screen @imagine-hussain ?
> 
> So the positioning is dependent on your screen width so it looks good sometimes and a bit off at other times. I'm happy to merge this as is though, and then leave the responsiveness to next sprint. But if @mck-jacky wants to address this, they Sgtm too.
> 
> The button styling I think looks off mainly cos of the default border that's still lingering around. @InvalidHero

sorry got no time no fix it in the past few days. I will fix it in the next sprint

**Pull request merged**